### PR TITLE
New SLE 15 rule set_nftables_new_connections

### PR DESCRIPTION
**Pull request by rumch-se**
#### Description:

- _New SLE 15 rule which covers CIS 3.5.2.7 Ensure outbound and established connections are configured_

#### Rationale:

- At the moment CIS profile for SLE 15 does not include this recommendation. The control cannot be automated,  and should be addressed manually.

### CODE REVIEW

File: linux_os/guide/system/network/network-nftables/set_nftables_new_connections/rule.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+references:
+    cis@sle15: 3.5.2.7
+
+ocil_clause: 'All nftables rules for established incoming, and for new and outbound connections do not match site policy'
+
+ocil: |-
+    To verify that all nftables rules on your system for established incoming connections match site policy,
+    run the following command:
+    <pre>$ sudo  # nft list ruleset | awk '/hook input/,/}/' | grep -E 'ip protocol (tcp|udp|icmp) ct state'</pre>
```
Comment (marcusburghardt) [inline linux_os/guide/system/network/network-nftables/set_nftables_new_connections/rule.yml:27]: It seems this this `#` is not expected in `sudo # nft`.

**Comment by rumch-se** [comment]
Hello @marcusburghardt 
Thank you for your feedback.
I have made expected corrections.
Have a nice day
Rumen

**Review by marcusburghardt — APPROVED** [review]

**Comment by marcusburghardt** [comment]
Overriding the CODEOWNERS protection since there is no SUSE contributor with write permission at the moment.

**Pull request merged**